tests/dummy_qrc/zurich.py

```diff
@@ -17,7 +17,7 @@
 )
 
 RUNCARD = pathlib.Path(__file__).parent / "zurich.yml"
-FOLDER = pathlib.Path(__file__).parent / "iqm5q/"
+FOLDER = pathlib.Path(__file__).parent / "zurich/"
```

Comment:
```suggestion
FOLDER = pathlib.Path(__file__).parent / "zurich"
```
If you avoid explicit separators (i.e. `"/"`) it is portable even to Windows (not that anyone really cares here...).